main/src/cgeo/geocaching/Image.java

```diff
@@ -15,27 +17,92 @@
 
 import java.io.File;
 
+/**
+ * Represent an Image along with Title and Description.
+ * Mostly used for representing image in/from logs.
+ */
 public class Image implements Parcelable {
-    private final String url;
-    private final String title;
-    private final String description;
+    @NonNull private Uri uri;
```

Comment:
Is a `Image.Builer.buildFrom(final Image image)` a good idea as a two lines replacement of actual setters?

The inner class don't have access to privates variables so getters will be necessary for all `Image` privates variables.
If this is ok, I imagine some class will have also protected variables not exposed via getters, so we can't copy the original `Image` entirely. That's a different comportement from just a setter in `Image`.